### ISSUE TITLE
Cannot run tests

### ISSUE THREAD

**Issue by chiniforooshan** [gvlproject/cloudbridge#15]
I'm not sure if there is anything wrong with my local setup or if this is a bug: when I run "tox -e py27" on master I get the following error:

>ERROR: invocation failed, logfile: /home/chiniforooshan/github/chiniforooshan/cloudbridge/.tox/py27/log/py27-1.log
ERROR: actionid=py27
msg=getenv
cmdargs=[local('/home/chiniforooshan/github/chiniforooshan/cloudbridge/.tox/py27/bin/pip'), 'install', '--pre', '-rrequirements.txt', 'coverage']
env={...}
Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'
Storing debug log for failure in /home/chiniforooshan/github/chiniforooshan/cloudbridge/.tox/py27/tmp/pseudo-home/.pip/pip.log
>
>ERROR: could not install deps [-rrequirements.txt, coverage]
>___________________________________________________________________________________ summary ____________________________________________________________________________________
ERROR:   py27: could not install deps [-rrequirements.txt, coverage]

cat .tox/py27/tmp/pseudo-home/.pip/pip.log

>Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'
Exception information:
Traceback (most recent call last):
  File "/home/chiniforooshan/github/chiniforooshan/cloudbridge/.tox/py27/local/lib/python2.7/site-packages/pip/basecommand.py", line 122, in main
    status = self.run(options, args)
  File "/home/chiniforooshan/github/chiniforooshan/cloudbridge/.tox/py27/local/lib/python2.7/site-packages/pip/commands/install.py", line 262, in run
    for req in parse_requirements(filename, finder=finder, options=options, session=session):
  File "/home/chiniforooshan/github/chiniforooshan/cloudbridge/.tox/py27/local/lib/python2.7/site-packages/pip/req.py", line 1546, in parse_requirements
    session=session,
  File "/home/chiniforooshan/github/chiniforooshan/cloudbridge/.tox/py27/local/lib/python2.7/site-packages/pip/download.py", line 278, in get_file_content
    raise InstallationError('Could not open requirements file: %s' % str(e))
InstallationError: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'

ls
> CHANGELOG.rst  cloudbridge  cloudbridge.egg-info  docs  LICENSE  README.rst  **requirements.txt**  setup.py  temp  test  tox.ini

**Comment by afgane** [comment]
Hmm - I just did a fresh clone of the repo and tests got under way, so it might be something with the setup. What system are you on?

Here's the exact set of commands I ran:
```
mkdir cb && cd cb
virtualenv -p /usr/bin/python2.7 .vnv
source .vnv/bin/activate
git clone git@github.com:gvlproject/cloudbridge.git && cd cloudbridge
pip install -e .[dev]
tox -e py27
```
At this point tests started running (but without [additional credentials setup](http://cloudbridge.readthedocs.org/en/latest/topics/testing.html?highlight=testing#id1), most of them failed).
